nrf_modem/include/nrf_socket.h

```diff
@@ -527,6 +623,10 @@ typedef uint32_t nrf_fd_set;
  * it can indicate that by setting this socket option before the next send call.
  */
 #define NRF_SO_RAI_WAIT_MORE               54
+/** Configurable TCP server session timeout in minutes.
+ * Range is 0 to 135. 0 is no timeout and 135 is 2 h 15 min. Default is 0 (no timeout).
+ */
+#define NRF_SO_TCP_SRV_SESSTIMEO           55
```

Comment:
Is this inactivity timeout? Or will the server shut the connection down regardless of the ongoing traffic after specified period?

Reply:
Also what socket level option is this? `SOL_SOCKET` or `IPPROTO_TCP`?

Reply:
It's an inactivity timeout, `SOL_SOCKET` level

Reply:
Thanks for clarifying